.github/CODEOWNERS

```diff
@@ -88,3 +90,21 @@
 /model/deployment/                @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-cicd-approvers
 /model/test/                      @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-cicd-approvers
 /model/vcs/                       @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-cicd-approvers
+
+# Database semantic conventions
+/docs/database/                   @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-db-approvers
```

Comment:
note to reviewers: we'll need to give write access to all new teams if this PR is accepted

Reply:
Is there a list of the new teams and their members somewhere?

Reply:
> Is there a list of the new teams and their members somewhere?

I believe these are all existing teams, we just forgot to add them to the CODEOWNERS file

Reply:
added write access to @open-telemetry/semconv-db-approvers @open-telemetry/semconv-messaging-approvers @open-telemetry/weaver-maintainers @open-telemetry/semconv-feature-flag-approvers , so GH now considers this file valid.